roles/brew/tasks/main.yml

```
@@ -0,0 +1,52 @@
+---
+- name: Install curl
+  package:
+    name: curl
+    state: present
+    update_cache: yes
+
+- name: Install git
+  package:
+    name: git
+    state: present
+    update_cache: yes
+
+- name: Check if brew installed
+  shell: "source ~/.bash_profile && brew --help"
+  register: brew_installed
+  changed_when: false
+  ignore_errors: true
+
+- name: Run brew installer script
+  shell: bash -l -c "curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh | bash"
```

```suggestion
  shell: |
    set -o pipefail 
    bash -l -c "curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh | bash"
```

I think this would solve the linter issue 🧐